Libplanet.Explorer/Queries/ExplorerQuery.cs

```diff
@@ -85,6 +85,12 @@ public ExplorerQuery(IBlockChainContext chainContext)
             foreach (var index in indexList)
             {
                 var block = store.GetBlock(index.value);
+                if (block is null)
+                {
+                    var message = $"Cannot found block: '{index.value}' in store.";
+                    throw new InvalidOperationException(message);
+                }
```

Comment:
The main purpose of variable `block` is to be assigned `Block`. Nullable declaration is unnecessary.

```suggestion
                Block block = store.GetBlock(index.value)
                    ?? throw new InvalidOperationException(
                        $"Could not find block with block hash {index.value} in store.");
```